Common/cpp/AnimatedSensor/AnimatedSensorModule.cpp

```diff
@@ -51,13 +53,13 @@ jsi::Value AnimatedSensorModule::registerSensor(
           value.setProperty(rt, "yaw", newValues[4]);
           value.setProperty(rt, "pitch", newValues[5]);
           value.setProperty(rt, "roll", newValues[6]);
-          handler.call(rt, value);
+          handler.call(rt, value, orientationDegrees);
```

Comment:
Ok, I can do this but what about the user's API? Should I make the orientation as separate shared value or put it in the `Value3D` and `ValueRotation`? I opt for separate shared value, because interface orientation is not sensor data. And if so, should I create a new ts type here just for data here?
```js
 (data, orientationDegrees) => {
        'worklet';
        if (ref.current.config.adjustToInterfaceOrientation) {
          if (sensorType === SensorType.ROTATION) {
            data = adjustRotationToInterfaceOrientation(
              data as ValueRotation,
              orientationDegrees
            );
```
will change into:
```js
(data) => {
        'worklet';
        if (ref.current.config.adjustToInterfaceOrientation) {
          if (sensorType === SensorType.ROTATION) {
            data = adjustRotationToInterfaceOrientation(
              data as ValueRotation+Orientation, //?
            );
```
now it doesn't look simpler unless it's just for optimisation.

Reply:
Yeah, generally we try to optimize the number of shared values, but in case of `useAnimatedSensor` it doesn't really make that much difference as we expect our users to call it only once within a component. Let's ask @piaskowyk who should know better.

Reply:
I asked him and he agreed that we should put orientation as property in sensor data shared value